crates/biome_css_formatter/src/css/lists/relative_selector_list.rs

```diff
@@ -23,7 +23,27 @@ impl FormatRule<CssRelativeSelectorList> for FormatCssRelativeSelectorList {
             // For example, a selector like `div span a` is structured like
             // `[div, [span, [a]]]`, so `a` would end up double-indented if it
             // was handled by the selector rather than here.
-            joiner.entry(&group(&indent(&formatted)));
+            let mut has_comments_before = false;
+            if let Some(relative_selector) = formatted.node()?.as_css_relative_selector() {
+                if let Some(computed_selector) =
+                    relative_selector.selector()?.as_css_compound_selector()
+                {
+                    if let Some(simple_selector) = computed_selector.simple_selector() {
+                        if let Some(type_selector) = simple_selector.as_css_type_selector() {
+                            let value_token = type_selector.ident()?.value_token()?;
+                            if value_token.has_leading_comments() {
+                                has_comments_before = true;
+                            }
+                        }
+                    }
+                }
+            }
+
+            if has_comments_before {
+                joiner.entry(&group(&formatted));
+            } else {
+                joiner.entry(&group(&indent(&formatted)));
```

Comment:
You should move the previous comment here I think, or you should update the comment that explains the new logic

Reply:
updated~